tectonic/dpx-pdfobj.c

```diff
@@ -1693,8 +1693,9 @@ apply_filter_TIFF2_1_2_4 (unsigned char *raster,
                 }
             }
         }
-        if (outbits > 0)
+        if (outbits > 0) {
             raster[k] = (outbuf << (8 - outbits)); k++;
```

Comment:
How did that happen? In `xetex.web` the line is `cur_val1:=cur_val1 + cur_val * @"200000;`